projects/rocprofiler-sdk/source/lib/python/rocpd/source/perfetto.cpp

```diff
@@ -281,6 +283,13 @@ write_perfetto(
             ::perfetto::TrackEvent::SetTrackDescriptor(_track, _desc);
 
             thread_tracks.emplace(itr.tid, _track);
+
+            auto _sampling_track = ::perfetto::Track::Global(++global_track_index);
```

Comment:
```suggestion
            auto _sampling_track = ::perfetto::Track{get_hash_id(fmt::format("sampling_{}", itr.tid)), this_pid_track};
```
Instead of using global_track_index, I used a hash, and we can get rid of the "Global Track Events" section in the perfetto.

Reply:
Done.